ngraph/core/src/pass/constant_folding_select.cpp

```diff
@@ -65,7 +68,7 @@ void pass::ConstantFolding::construct_constant_select()
         element::i64, Shape{2, 3, 4}, pattern::has_class<op::Constant>());
     auto f_label = make_shared<pattern::op::Label>(
         element::i64, Shape{2, 3, 4}, pattern::has_class<op::Constant>());
-    auto select_v0_op = make_shared<op::v0::Select>(selection_label, t_label, f_label);
+    auto select_v0_op = make_shared<op::v1::Select>(selection_label, t_label, f_label);
```

Comment:
duplication. please remove select matching completely

Reply:
Removed, thanks

Reply:
Sorry if I'm missing something, but what this matcher duplicates? Select OP doesn't have evaluate interface implementation and as far as I understand can't be folded in a different way (correct me if I'm wrong).